var/spack/repos/builtin/packages/openexr/package.py

```diff
@@ -6,14 +6,14 @@
 from spack import *
 
 
-class Openexr(AutotoolsPackage):
+class Openexr(CMakePackage):
     """OpenEXR Graphics Tools (high dynamic-range image file format)"""
 
     homepage = "http://www.openexr.com/"
-    url = "https://github.com/openexr/openexr/releases/download/v2.3.0/openexr-2.3.0.tar.gz"
+    url      = "https://github.com/AcademySoftwareFoundation/openexr/archive/v2.5.3.tar.gz"
```

Comment:
spacing after variable name seems wrong (s/b single space)

Reply:
When a boilerplate Spack package is created it comes with the extra spaces leading to the '=' symbols being aligned. I added the spaces to match that precedence. However, I can revert it to only one space if you think it's appropriate.